chrome-finder.ts

```diff
@@ -150,10 +152,23 @@ export function linux() {
   return sort(uniq(installations.filter(Boolean)), priorities);
 }
 
+export function wsl() {
+  // Manually populate the environment variables assuming it's the default config
+  const user = findWSLUser();
+
+  process.env.LOCALAPPDATA = `/mnt/c/Users/${user}/AppData/Local`;
+  process.env.PROGRAMFILES = '/mnt/c/Program Files';
+  process.env['PROGRAMFILES(X86)'] = '/mnt/c/Program Files (x86)';
+
+  return win32();
+}
+
 export function win32() {
   const installations: Array<string> = [];
+  // Because we could be in WSL, we need to make sure to use the right separator
   const suffixes = [
-    '\\Google\\Chrome SxS\\Application\\chrome.exe', '\\Google\\Chrome\\Application\\chrome.exe'
+    `${path.sep}Google${path.sep}Chrome SxS${path.sep}Application${path.sep}chrome.exe`,
```

Comment:
kinda awkward. how about using `path.join()` instead?

Reply:
Sorry if I'm missing some context, but windows will happily use `\` or `/` as directory separators (it's only path separators (`:` vs `;`) that are different). So this might not be necessary?

Reply:
That was my initial though but it didn't work. If I don't change things I end up with the following paths:
```
/mnt/c/Users/antonmo/AppData/Local/\Google\Chrome SxS\Application\chrome.exe
/mnt/c/Users/antonmo/AppData/Local/\Google\Chrome\Application\chrome.exe
/mnt/c/Program Files/\Google\Chrome SxS\Application\chrome.exe
/mnt/c/Program Files/\Google\Chrome\Application\chrome.exe
/mnt/c/Program Files (x86)/\Google\Chrome SxS\Application\chrome.exe
/mnt/c/Program Files (x86)/\Google\Chrome\Application\chrome.exe
```

and no installations are found:
```
(node:3007) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): Error: No Chrome Installations Found
(node:3007) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
I tried doing `path.normalize` when joining with the prefix (`const chromePath = path.normalize(path.join(prefix, suffix));`) but the result was still the same (at least on 8.4.0) :(